apis/Google.Cloud.Asset.V1Beta1/Google.Cloud.Asset.V1Beta1/AssetServiceGrpc.cs

```diff
@@ -17,6 +17,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+//
```

Comment:
Slightly odd extra // here. Not that it matters.

Reply:
This is gRPC generation, not veneer...

Reply:
Oops, you're right. I'm not going to worry about it for now :)